locales/it/translation.json

```diff
@@ -0,0 +1,183 @@
+        },
+        "page": {
+            "hero": {
+                "description": "Un editor di testo open source e moderno, creato per il web.",
+                "download": "Scarica Brackets <d id='download-brackets-version'>1.3</d>",
+                "bundle-info": "+ Extract (Anteprima) by Adobe",
+                "os-alert": {
+                    "general": "Sfortunatamente, __os__ non è supportato. <a href='__url__'>Controlla i Requisiti.</a>",
+                    "win-xp": "Sfortunatamente, Window XP non è supportato. Leggi <a href='__url__'>questo post</a> per avere altre informazioni.",
+                    "debian-7": "Sfortunatamente, Debian 7 non è supportato. Controlla questo <a href='__url__'>issue su GithHub</a> per avere altre informazioni."
+                },
+                "non-bundle-download": "Scarica Brackets senza Extract",
+                "other-downloads": "Altri Downloads"
+            },
+            "content": {
+                "blurb": "Con strumenti visuali e supporto ai preprocessori, Brackets è un moderno editor di testo che rende semplice programmare per il web. Prova Creative Cloud Extract (in anteprima) per Brackets: un modo facile per ottenere del codice CSS pulito e minimale, direttamente da un PSD e senza dover scriverene una sola linea!",
+                "why": {
+                    "header": "Perché Usare Brackets?",
+                    "content": "Brackets è un editor potente, moderno e leggero. Abbiamo aggiunto strumenti visuali nell'editor così da poter ricevere il giusto aiuto quando se ne ha bisogno. Con nuove funzionalità ed estensioni rilasciate ogni 3-4 settimane, è come ricevere regali tutto l'anno!"
+                },
+                "extract": {
+                    "header": "Ricevi suggerimenti da un PSD",
+                    "content": "L'estensione Extract per Brackets (Anteprima) to permette di ricavare gli stili di un PSD: colori, fonts, sfumature e altre informazioni, per scrivere del CSS pulito e minimale tramite i suggermenti del codice. Puoi anche estrarre livelli come immagini, usare le informazioni del PSD per definire variabili per i processori e misurare facilmente la distanza tra gli oggetti. Tutto con il codice sempre sotto gli occhi.",
+                    "more": "Altro …",
+                    "download": "Scarica Brackets + Extract (Anteprima) by Adobe"
+                },
+                "feature-highlights": {
+                    "inline-editor": {
+                        "header": "Editor Inline",
+                        "content": "Anziché spostarsi da un file all'altro, Brackets permette di aprire una finestra nel codice che ti interessa. Vuoi lavorare sul CSS applicato ad uno specifico ID? Posiziona il cursore su quell'ID, premi <kbd>Command</kbd> / <kbd>Ctrl+E</kbd> e Brackets mostrerà tutti i selettori CSS con quell'ID in una finestra nel il codice, così che tu possa lavorare facilmente su diversi file del tuo progetto."
```

Comment:
Review comment per @Denisov21:
Drop the `nel`: `in una finestra nel il codice` -> `in una finestra il codice`